src/main/java/org/example/MigrationConfig.java

```diff
@@ -95,7 +98,10 @@ public Step step(StepBuilderFactory stepBuilderFactory,
 		// marklogic-spring-batch component that is used to write a Spring ColumnMap to an XML or JSON document
 		ColumnMapSerializer serializer = null;
 		if (documentType != null && documentType.toLowerCase().equals("json")) {
-			serializer = new JacksonColumnMapSerializer();
+			ObjectMapper objectMapper = new ObjectMapper();
+			objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```

Comment:
I'm sure this would be applicable to some projects, but is it going to be universally true for all projects? This is an area where I don't think ml-migration-starter should get into - i.e. customizing how the JSON is serialized.

Reply:
well.. you could argue that the target is going to be MarkLogic, and if you like to index them as dates, they need to be in ISO format.. :-)

but fair point, a flag to toggle this behavior would be the least..